test/access.jl

```diff
@@ -64,10 +65,11 @@
         @test iterate(RNAKmer("ACUG"), 4) == (RNA_G, 5)
         
 
-        @test iterate(RNAKmer("ACUG"), 1)  !== nothing
+        @test !isnothing(iterate(RNAKmer("ACUG"), 1))
         @test iterate(RNAKmer("ACUG"), 4)  !== nothing
         @test iterate(RNAKmer("ACUG"), 5)  === nothing
-        @test_throws BoundsError iterate(RNAKmer("ACUG"), -1)
+        @test iterate(RNAKmer("ACUG"), -1) === nothing
+        @test iterate(RNAKmer("ACUG"), 0) === nothing
```

Comment:
Can we update all of these? My apologies - I didn't notice before that you were copying the style of tests that were already there. I think either revert my suggestion above, or change them all.